scripts/telemetryui_update_materialized_views.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Update telemetryui views
+
+[Service]
+Type=simple
+User=telemetry
+Group=telemetry
+WorkingDirectory=@@install_path@@/scripts
+ExecStart=@@install_path@@/venv/bin/python3 @@install_path@@/scripts/update_materialized_views.py --password @@db_password@@
```

Comment:
Since the code expects the install_path to be set, I think you should update the deploy.sh to install these service and timer files.